addons/account_peppol/models/account_edi_format.py

```diff
@@ -0,0 +1,89 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import _, models, SUPERUSER_ID
+from odoo.exceptions import UserError
+
+from odoo.addons.account.models.company import PEPPOL_DEFAULT_COUNTRIES
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountEdiFormat(models.Model):
+    _inherit = 'account.edi.format'
+
+    def _get_proxy_identification(self, company):
+        if self.code != 'peppol':
+            return super()._get_proxy_identification(company)
+        if not company.peppol_eas or not company.peppol_endpoint:
+            raise UserError(_("Please fill in the EAS code and the Participant ID code."))
+        return f'{company.peppol_eas}:{company.peppol_endpoint}'
+
+    def _get_peppol_builder(self, company):
+        if self.code == 'ubl_bis3':
+            return self.env['account.edi.xml.ubl_bis3']
+        if self.code == 'nlcius_1':
+            return self.env['account.edi.xml.ubl_nl']
+        if self.code == 'ubl_de':
+            return self.env['account.edi.xml.ubl_de']
+        if self.code == 'ubl_a_nz':
+            return self.env['account.edi.xml.ubl_a_nz']
+        if self.code == 'ubl_sg':
+            return self.env['account.edi.xml.ubl_sg']
```

Comment:
I think we should remove those two completely from the PR.

- we don't allow to enable Peppol for AU & NZ yet, even in master
- BIS 3.0 AUNZ is actually deprecated in favor of PINT AUNZ

Therefore the only 3 formats supported at sending so far on Peppol are BIS3.0, NLCIUS and xRechnung.